Library/Homebrew/missing_formula.rb

```diff
@@ -15,7 +15,7 @@ def blacklisted_reason(name)
         EOS
         when "tex", "tex-live", "texlive", "latex" then <<~EOS
           Installing TeX from source is weird and gross, requires a lot of patches,
-          and only builds 32-bit (and thus can't use Homebrew dependencies)
+          and only builds 32-bit (and thus can’t use Homebrew dependencies)
```

Comment:
Please change this back, thanks!

Reply:
Sure. I only included this because github's syntax highlighter seems to choke on the single quote.